docs/api/users-get-all-users.md

```diff
@@ -50,3 +50,7 @@ None
 | ------------- | ----------- | ----------- |
 | 200 | Success | Requested data returned successfully |
 |  ECONNREFUSED | N/A | Service is offline. Start the service and try again. |
+
+### Additional Links
+
+[List all user last names](api/list_all_user_last_names.md)
```

Comment:
```suggestion
[List all user last names](list_all_user_last_names.md)
```
The URL in your file is a relative URL, so the link that results from your code is:

```
https://rbwatson.github.io/to-do-service-public/api/api/list_all_user_last_names.md
```

If the file that you're linking to is in the same folder as you're linking from, all you need to put is the filename.